### DISCUSSION TITLE
分享短链&二维码 下载链接一键提取 这两个功能是否还没有真正实现？

### DISCUSSION THREAD

**Discussion by ljm12914** [libsgh/PanIndex#151] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/29831474/137489981-08dc48ef-9f82-4c3d-892b-3f0e0ec4a806.png)

按“复制全部下载链接”没有反应，按“分享”也没有反应

**Comment by libsgh** [comment, marked as answer]
这两个都已经实现，可能是js有缓存 浏览器CTRL+F5刷新试试，复制全部下载链接没有反应可能是当前目录没有文件都是目录，没有可复制的文件链接

**Reply by ljm12914** [reply]
问题出在Google Chrome的弹出窗口拦截功能
